src/test/unit_tests/test_namespace_fs.js

```diff
@@ -459,6 +462,11 @@ mocha.describe('namespace_fs', function() {
         const restore_key = 'restore_key_1';
         const data = crypto.randomBytes(100);
 
+        mocha.before(async function() {
+            const dir = await mkdtemp(`${os.tmpdir()}${path.sep}`);
+            config.NSFS_WAL_DIR = dir;
```

Comment:
why is this needed? can't we mkdir in nsfs instead of relying on someone to create it for us?